motoman_driver/MotoPlus/MotionServer.c

```diff
@@ -31,6 +31,10 @@
 
 #include "MotoROS.h"
 
+#ifdef DEBUG

```

Comment:
I recommend removing this condition. The `DEBUG` preprocessor was to protect against costly `printf` commands. The udp-broadcast method is way more efficient. I think it's safe to allow these w/o also enabling the `printf`.